paperoni/papers2.py

```diff
@@ -38,7 +38,8 @@ class Topic:
 class Author:
     links: Sequence[Link] = ()
     name: str = None
-    affiliations: Sequence[str] = ()
+    affiliations: Sequence[str] = ("",)
+    role: str = None
```

Comment:
An author can take on multiple roles through time, and possibly multiple roles at the same time, so it shouldn't be represented this way. We can leave it out for this PR.